src/js/api/services/ProductSupplierApi.js

```diff
@@ -5,7 +5,10 @@ import {
 import apiClient from 'utils/apiClient';
 
 export default {
-  getPreferenceTypeOptions: (config) => apiClient.get(PREFERENCE_TYPE_OPTIONS, config),
+  getPreferenceTypeOptions: (config) => apiClient.get(
+    PREFERENCE_TYPE_OPTIONS('?includeMultiple=true&includeNone=true'),
+    config,
+  ),
```

Comment:
I think you should pass those params within the `config` object
```
params: {
   includeMultiple: true,
   includeNone: true,
}
```

Reply:
fixed